client/layout/guided-tours/docs/API.md

```diff
@@ -0,0 +1,162 @@
+# Guided Tours API
+
+Guided Tours are declared as a tree of JSX components.
```

Comment:
Since
- JSX is just syntactic sugar;
- I believe these are technically elements (as in: `Foo` is a component, `<Foo />` is an element);

I think we could rephrase as

```diff
-Guided Tours are declared as a tree of JSX components.
+Guided Tours are declared in JSX as a tree of elements.
```